relay-profiling/src/transaction_metadata.rs

```diff
@@ -13,9 +13,17 @@ pub struct TransactionMetadata {
     #[serde(default, deserialize_with = "deserialize_number_from_string")]
     pub active_thread_id: u64,
 
-    #[serde(deserialize_with = "deserialize_number_from_string")]
+    #[serde(
+        default,
```

Comment:
To clarify, my above comment regards this change. You cannot make this field optional in SDKs if they have been sending this before. Otherwise, your SDK upgrade will break self-hosted (and temporarily single tenant) Sentry installations. Parsing of those payloads will fail, and (old) Relay will stop ingesting payloads from these newer SDK versions.

Reply:
Please disregard this as per https://github.com/getsentry/relay/pull/1878#issuecomment-1446521577.